README.md

```diff
@@ -63,15 +63,20 @@ Want to add your own project to this list? It's easy: check out
   for building on Raspberry Pi. Needs an owner.
 * [**WSL2 GPU Guide**](wsl2_gpu_guide): Instructions for enabling GPU with Tensorflow
   on a WSL2 virtual machine.
+  
 
 ### WIP / Other
 
+* [**Build Stats Functions**](utils/build_stats_functions): Cloud Functions
+  for analyzing Bazel build profiles and build event protocols and uploading them
```

Comment:
The path here needs to be fixed, and the description needs to match the description in the README (see CONTRIBUTING.md and the template files).